### PR TITLE
Task 105 - Story Completion

### PR DESCRIPTION
**Pull request by Mirzyaaliii**
Please review.

### CODE REVIEW

**Comment by danyaljj** [comment]
Thanks! 

I have a bit issue with the meaning of "complete"/"incomplete" in this context. What does it mean for a story to be "incomplete"? Take these four sentences, for example: 

```
Sentence1: It was  my final performance in marching band. Sentence2: I was playing the snare drum in the band. Sentence3: We played Thriller and Radar Love. Sentence4: The performance was flawless.
```
Sounds like a "complete" to me, even without a 5th sentence. All that to say that, I would avoid using the phrasing of "complete"/"incomplete".

**Comment by Mirzyaaliii** [comment]
> Thanks!
> 
> I have a bit issue with the meaning of "complete"/"incomplete" in this context. What does it mean for a story to be "incomplete"? Take these four sentences, for example:
> 
> ```
> Sentence1: It was  my final performance in marching band. Sentence2: I was playing the snare drum in the band. Sentence3: We played Thriller and Radar Love. Sentence4: The performance was flawless.
> ```
> 
> Sounds like a "complete" to me, even without a 5th sentence. All that to say that, I would avoid using the phrasing of "complete"/"incomplete".

I corrected the definition, please review again.

File: tasks/task105_story_completion.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "Contributors": "Mirali Purohit",
     "Categories": "Sentence Generation",
-    "Definition": "In this task, you're given a story which is incomplete. Each story has four sentences written in natural language. Your job is to complete the end part of an incomplete story by predicting appropriate last sentence.",
+    "Definition": "In this task, you're given a four sentences of story written in natural language. Your job is to complete end part of the story by predicting appropriate last sentence.",
```
Comment (danyaljj) [inline tasks/task105_story_completion.json]: add something along this line: .... in a manner that is coherent with the provided sentences (?)

Comment (Mirzyaaliii) [reply tasks/task105_story_completion.json]: Please review again.

**Review by swarooprm — CHANGES_REQUESTED** [review]
Thanks for creating the PR! Seems like the test is failing because there is instance repetition, you can remove duplicate instances in your task. Also, you need to add an updated readme file (similar to any other PR where tests are passing e.g. https://github.com/allenai/natural-instructions-expansion/pull/35), else the test might fail again.

**Comment by Mirzyaaliii** [comment]
Please review again.

**Comment by danyaljj** [comment]
Thanks! Please pull and merge with the main branch. That should trigger the automatic tests.

**Comment by Mirzyaaliii** [comment]
To resolve a conflict and some other errors, I am closing this pull request and creating a new one with the same task.

**Pull request closed without merging**